docs/content/schemas.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Type Checking
+kind: misc
+weight: 2
+---
+
+## Using schemas to enhance the Rego type checker
+
+You can provide an input schema to `opa eval` to improve static type checking and get more precise error reports as you develop Rego code.
+The `-s` flag can be used to upload a single schema for the input document in JSON Schema format.
```

Comment:
Nit: since we'll want to rename this flag to avoid future naming conflicts, let's not include the short version of the flag.

Reply:
Same as above comment ^^.